src/main/java/io/discloader/discloader/core/entity/guild/GuildMember.java

```diff
@@ -108,11 +108,7 @@ public GuildMember(IGuildMember member) {
 		user = member.getUser();
 		guild = member.getGuild();
 		nick = member.getNickname();
-		roleIDs = new String[member.getRoles().size()];
-		int i = 0;
-		for (IRole role : member.getRoles().values()) {
-			roleIDs[i] = role.getID();
-		}
+		roleIDs = member.getRoles().values().toArray(new String[] { });
```

Comment:
this won't work. needs to be `member.getRoles().keySet().toArray(new String[] {});`